core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/azureModels.scala

```diff
@@ -36,5 +36,6 @@ final case class LandingZoneResources(landingZoneId: UUID,
                                       batchNodesSubnetName: SubnetworkName,
                                       aksSubnetName: SubnetworkName,
                                       region: com.azure.core.management.Region,
-                                      applicationInsightsName: ApplicationInsightsName
+                                      applicationInsightsName: ApplicationInsightsName,
+                                      postgresName: Option[PostgresName]
```

Comment:
Added `PostgresName` to the landing zone resources. This PR doesn't actually use it -- I'm just anticipating it will be needed when we start passing the postgres info to the apps. It is optional to handle older landing zones which don't have postgres flex yet.